assets/js/app/LocusZoom.js

```diff
@@ -15,35 +15,48 @@
     // Create a new instance by instance class and attach it to a div by ID
     // NOTE: if no InstanceClass is passed then the instance will use the Intance base class.
     //       The DefaultInstance class must be passed explicitly just as any other class that extends Instance.
-    exports.addInstanceToDivById = function(InstanceClass, id){
+    exports.addInstanceToDivById = function(id, datasource, layout, state){
```

Comment:
Ah, i had removed the test but it got re-added when i merged with master at one point.